### PR TITLE
No JIRA/BZ. Updates the node observability section

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
Version(s): 4.11+

Issue: No JIRA/BZ. I submitted Node Observability Operator doc for a crowd sourced minimalism session. The doc is updated based on the suggestions provided by panel to improve the customer focus, readability, and remove fluff in the doc.

Link to docs preview: https://50699--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/node-observability-operator.html

Additional information: @alebedev87 and @sunilcio PTAL and let know your feedback.

### CODE REVIEW

File: modules/node-observability-high-level-workflow.adoc [hunk 1]
```diff
@@ -4,8 +4,10 @@
...
-= High level workflow of the Node Observability Operator
+= Workflow of the Node Observability Operator
 
-After you install the Node Observability Operator in the {product-title} cluster, you have to create a `NodeObservability` custom resource, which creates a DaemonSet to deploy a Node Observability agent on each worker node.
+The following workflow outlines on how to query the profiling data using the Node Observability Operator:
 
-To request a profiling query, you have to create a `NodeObservabilityRun` resource that requests the deployed Node Observability agent to trigger the CRI-O and Kubelet profiling. After the profiling is completed, the Node Observability agent stores the profiling data inside the container file system `/run/node-observability` directory, which is available for query.
+. Install the Node Observability Operator in the {product-title} cluster.
+. Create a NodeObservability custom resource to enable the CRI-O profiling on the worker nodes of your choice.
+. Run a profiling query to retrieve the profiling data.
```
Comment (alebedev87) [inline modules/node-observability-high-level-workflow.adoc:13]:
```suggestion
. Run a profiling query to generate the profiling data.
```

**Review by alebedev87 — COMMENTED** [review]
LGTM, just a small comment.

File: modules/node-observability-high-level-workflow.adoc (hunk 1 above)
Comment (darshan-nagaraj) [reply modules/node-observability-high-level-workflow.adoc]: fixed

File: modules/node-observability-run-profiling-query.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@
 [id="running-profiling-query_{context}"]
 = Running profiling query
 
-Profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. The Node Observability Operator stores the profiling data inside the container file system `/run/node-observability` directory. To request profiling data query, you have to create a `NodeObservabilityRun` resource.
+To run profiling query, you must create a `NodeObservabilityRun` resource. The profiling query is a blocking operation that fetches CRI-O and Kubelet profiling data for a duration of 30 seconds. When the profiling query is complete, you must retrieve the profiling data inside the container file system `/run/node-observability` directory.
```
Comment (abrennan89) [inline modules/node-observability-run-profiling-query.adoc:9]: >To run profiling query

Should this be "a profiling query" or "the profiling query"?

>When the profiling query is complete

Suggest "After the profiling..."

File: modules/node-observability-run-profiling-query.adoc [hunk 3]
```diff
@@ -33,7 +33,7 @@ spec:
     name: cluster
 ----
 
-. Run the `NodeObservabilityRun` to trigger the profiling:
+. Trigger the profiling by running the `NodeObservabilityRun`:
```
Comment (abrennan89) [inline modules/node-observability-run-profiling-query.adoc:36]:
```suggestion
. Trigger profiling by running the `NodeObservabilityRun` resource:
```

File: modules/node-observability-run-profiling-query.adoc [hunk 4]
```diff
@@ -65,9 +65,9 @@ conditions:
 ----
 
 +
-Profiling query is complete when the status is `True` and type is `Finished`.
+The profiling query is complete when the status is `True` and type is `Finished`.
```
Comment (abrennan89) [inline modules/node-observability-run-profiling-query.adoc:68]:
```suggestion
The profiling query is complete once the status is `True` and type is `Finished`.
```

File: scalability_and_performance/node-observability-operator.adoc [hunk 5]
```diff
@@ -1,17 +1,17 @@
 :_content-type: ASSEMBLY
 [id="using-node-observability-operator"]
-= Understanding the Node Observability Operator
+= Requesting CRI-O and Kubelet profiling data using the Node Observability Operator
```
Comment (abrennan89) [inline scalability_and_performance/node-observability-operator.adoc:3]:
```suggestion
= Requesting CRI-O and Kubelet profiling data by using the Node Observability Operator
```

**Review by abrennan89 — COMMENTED** [review]
Looks great, just a few minor comments / suggestions!

**Review by abrennan89 — COMMENTED** [review]
LGTM!

**Pull request merged**